setup.cfg

```diff
@@ -0,0 +1,21 @@
+[mypy]
+python_version = 3.7
+show_error_codes = True
+pretty = True
+strict = True
+files = src/isodate/*.py
+
+[tool:pytest]
+# https://github.com/pytest-dev/pytest/issues/1877
+filterwarnings =
+    ignore::pytest.PytestCollectionWarning
+
+[tool:black]
+line-length = 88
+
+[tool:isort]
+profile = black
+line_length = 88
```

Comment:
This is already included in the profile: https://pycqa.github.io/isort/docs/configuration/profiles.html
```suggestion
```

Reply:
5a9a092